src/CameraControls/CameraControls.tsx

```diff
@@ -224,7 +224,7 @@ export const CameraControls: FC<
       } else {
         cameraRef.current.mouseButtons.left = 2;
         cameraRef.current.mouseButtons.middle = 2;
-        cameraRef.current.mouseButtons.wheel = 16;
+        cameraRef.current.mouseButtons.wheel = ThreeCameraControls.ACTION.DOLLY;
```

Comment:
this is the same as `cameraRef.current.mouseButtons.wheel = 8`, but I figured this was clearer

could also update the above to
```
cameraRef.current.mouseButtons.left = ThreeCameraControls.ACTION.TRUCK;
cameraRef.current.mouseButtons.middle = ThreeCameraControls.ACTION.TRUCK;
```

Reply:
lets do that.

Reply:
oh also just found a bug where the camera's not disabled when `disabled` is true

I have a little fix, lmk if I should push it into here. It's not really related